geopyspark/geotrellis/catalog.py

```diff
@@ -250,6 +251,15 @@ def write(uri,
             each index. Meaning, what time intervals are used to seperate each record. While this is
             set to ``None`` as default, it must be set if saving spatial-temporal data.
             Depending on the indexing method chosen, different time units are used.
+        time_resolution (str or int, optional): Determines how data for each ``time_unit`` should be
```

Comment:
`time_resolution` sounds like a the in-house name for this, struggling to come up with a more intuitive alternative though. Fishing for ideas here.

Reply:
What about `temporal_grouping`? Since it's used to determine how the units of time should be grouped together when saved.

Reply:
Maybe `time_window` or `time_span` if those don't inspire clarity in you we can leave it as is.

Reply:
The only issue I have with those is that they could be interpreted to mean something different. Like, they control the the time frame of the data being saved (ie. only data falling within this window/span will be saved).